### ISSUE TITLE
Undeclared dependencies: binutils-dev and libiberty-dev

### ISSUE THREAD

**Issue by philpem** [NationalSecurityAgency/ghidra#502]
**Describe the bug**
Building in Eclipse results in build errors unless the `binutils` and `libiberty` development packages are installed.

Build fails at the `make -j4 decomp_dbg` stage and reports that either `bfd.h` or `-liberty` could not be found.

**To Reproduce**
1. Import project into Eclipse and leave to build automatically
2. See error

**Expected behavior**
Build succeeds

**Solution**
Declare in DevDocs.md that `libiberty-dev` and `binutils-dev` need to be installed in order to build Ghidra.

**Environment (please complete the following information):**
 - OS: Ubuntu 18.04 "bionic"
 - Java Version: 11.0
 - Ghidra Version: Git master

**Comment by Escapingbug** [comment]
+1 for running into error because of this. This seems to be needed by debug console only. And MacOS can fall into full pain to get the dependencies. I still have no idea how to do that exactly..

**Comment by philpem** [comment]
I've actually found that this is due to running the Gradle preparation in the wrong directory (I deleted my original clone, cloned Ghidra again into ~/git, but forgot to 'cd' to the new directory in my terminal).

Run this command line:
```bash
gradle prepDev -x yajswDevUnpack eclipse && gradle sleighCompile && gradle buildNatives_linux64
```
Then switch to Eclipse, right-click the "__root" project and select Gradle -> Sync. You might have to clean the project first.

I don't know where it gets BFD in that case (I've tried `gradle clean` followed by the above command line) but it certainly seems to work.

Regardless, there's likely no harm in having libiberty and the Binutils dev files installed.

**Comment by philpem** [comment]
> I've actually found that this is due to running the Gradle preparation in the wrong directory (I deleted my original clone, cloned Ghidra again into ~/git, but forgot to 'cd' to the new directory in my terminal).

Yeah okay... without the dev packages for BFD and Libiberty, it'll build on the command line (somehow) but fail to build inside Eclipse (the Decompiler build fails).

So yes, in summary, you do need these dev packages.

**Comment by nsadeveloper789** [comment]
It looks like the configuration of the CDT `_Decompiler` project in Eclipse is invoking the wrong target. `decomp_dbg` is a console-based debugging and testing version of the decompiler. If you want that version, then yes, you'll need those development packages. The Decompiler `Makefile` is only used for development anymore. 

To build for production, we use Gradle's native C/C++ support, which does not have anything to do with the `Makefile`. The production configuration does not contain the console/debug features. The equivalent make target is `ghidra_opt`. I suppose this issue could be fixed by changing the CDT project configuration in Eclipse.

**Comment by nsadeveloper789** [comment]
Fixed by 53510f1b0cbb28220baadcb5ced852ad4e0afbeb. Changed `_Decompiler` project configuration.
